### DISCUSSION TITLE
Navigation between hunks in CommitView

### DISCUSSION THREAD

**Discussion by dapc11** [NeogitOrg/neogit#796] [Q&A] [answered]
Hello!

I haven't been able to find any information regarding navigation in NeogitCommitView. It would be awesome to be able to jump between hunks in the same manner as in the status view. Is this possible? If yes, how?

Thanks in advance!

**Comment by dapc11** [comment, marked as answer]
Nevermind this question, I now see that the same navigation is working for commit view as well.

**Reply by CKolkey** [reply]
Yup :)
